### ISSUE TITLE
Try to install the pokemonGo-Bot on a Raspberry Pi

### ISSUE THREAD

**Issue by Chrisvdberg** [PokemonGoF/PokemonGo-Bot#3235]
Please check configuration at http://jsonlint.com/ before posting an issue.

When i run the pokecly.pi -> 

(PokemonGo-Bot) pi@doriangray:~/PokemonGo-Bot $ python pokecli.py 
Traceback (most recent call last):
  File "pokecli.py", line 38, in <module>
    from pgoapi.exceptions import NotLoggedInException, ServerSideRequestThrottlingException, ServerBusyOrOfflineException
ImportError: No module named pgoapi.exceptions

I think something goes wrong with pip install -r requirements.txt

I got a lot of error s like: 
 Failed building wheel for numpy
  Running setup.py clean for numpy
  Complete output from command /home/pi/PokemonGo-Bot/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-B73AeV/numpy/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" clean --all:
  Running from numpy source directory.
  
  `setup.py clean` is not supported, use one of the following instead:
  
    - `git clean -xdf` (cleans all files)
    - `git clean -Xdf` (cleans all versioned files, doesn't touch
                        files that aren't checked into the git repo)
  

And I got that a lot of times..

**Comment by jamosaur** [comment]
`sudo apt-get install python-dev`

run `pip install -r requirements` afterwards

**Comment by jrjhealey** [comment]
I've found that sometimes invoking pip with sudo (`sudo -H pip install -r requirements.txt`) resolves some issues for me.

**Comment by Didlid** [comment]
The way i had to get mine working was using virutalenv and then every time before running the bot do sources/activate

**Comment by xSomeFoolx** [comment]
I was having trouble installing on my RPi3 with Raspbian Jessie Lite as well.

Make sure you have virtualenv, python-pip, and python-dev installed also.

> sudo apt-get install virtualenv python-pip python-dev

When running the install.sh it did not go into virtualenv properly. I had to do it manually before running the install.

I ran "virtualenv ." then "source bin/activate" in the PokemonGo-Bot directory manually before running "sh install.sh".

This gave me a good install and took a bit to compile since the proc is not the fastest and it only uses one core at a time.
